cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -51,4 +52,25 @@ describe("Tab management", () => {
       cy.get('[data-cy="input-civil-time"] input').clear({ force: true }).type("123");
     });
   });
+
+  const testWorker = "Pielęgniarka 2";
+  describe("Editing workers time", () => {
+    it("properly handles worker hours edition", () => {
+      cy.get('[data-cy="exit-drawer"]').click();
+      cy.get(`[data-cy="edit-worker-${testWorker}"]`).click();
+      cy.get('[data-cy="contract-time-dropdown"]').click().get('[data-cy="half"]').click();
+      cy.get(`[data-cy="btn-save-worker"]`).click();
+      cy.get(`[data-cy="worker-hours-${testWorker}"]`).contains("umowa o pracę 1/2");
+      cy.get('[data-cy="btn-schedule-tab"]').click();
+      cy.checkHoursInfo({
+        teamIdx: 0,
```

Comment:
Move this (`teamIdx`, `workerIdx`) above, alongside `const testWorker` and reuse it for the additional test I mentioned above.

Reply:
And done. :)